### ISSUE TITLE
Is there any extra work to use batch norm and train by native tensorflow operations?

### ISSUE THREAD

**Issue by adamcavendish** [tflearn/tflearn#707]
I'd like to build network with the help of `tflearn` but train by native tensorflow operations, like [extending_tensorflow/layers.py](https://github.com/tflearn/tflearn/blob/master/examples/extending_tensorflow/layers.py).

Is there anything special to consider when using `batch_normalization` in network, or `sess.run([optimizer, loss], feed_dict={ ... })` would be just fine.
